rtest.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE testset SYSTEM "/usr/local/share/rtest.dtd">
```

Comment:
Is this hardcoding correct?

Reply:
That doesn't look windows friendly.   
You could also leave the dtd specification out of the rtest.xml for now and use a dtd for only an external validation process.

Reply:
For the moment, the DTD isn't used for validation of `rtest.xml`, and this doctype declaration was primarily used for my IDE and can safely be removed. Eventually we'd like to have the dtd at a static location and reference using a public doctype.